web/themes/custom/accesstheme/templates/page.html.twig

```diff
@@ -162,7 +162,7 @@ set footer_classes = ' ' ~
         </a>
       </div>
       <div class="col-md-6 text-center text-md-end mt-4">
-        <a href="https://identity.access-ci.org/aup.html">Acceptable Use</a> | <a href="/code-of-conduct">Code of Conduct</a> | <a href="/privacy-policy">Privacy Policy</a>
+        <a href="/acceptable-use/">Acceptable Use Policy</a> | <a href="/code-of-conduct">Code of Conduct</a> | <a href="/privacy-policy">Privacy Policy</a>
```

Comment:
Please link to the full url: https://access-ci.org/acceptable-use/

Reply:
Okay I will do that!